foundry.toml

```
@@ -0,0 +1,8 @@
+[profile.default]
+src = 'contracts'
+out = 'out'
+libs = ['node_modules', 'lib']
+test = 'test/foundry'
+cache_path  = 'forge-cache'
+
```

Adding remapping for OpenZeppelin contracts:

```suggestion
remappings = [
    "@openzeppelin/contracts-upgradeable/=lib/openzeppelin-contracts-upgradeable/contracts/",
    "openzeppelin-contracts/contracts/=lib/openzeppelin-contracts/contracts/"
]
```

---

But the problem is not having the dependencies from HH or not, the problem happens when I try to follow the documentation, it fails to execute the tests without having these remapping. So if there is a fix for it, please add it to the readmfile/foundry config.